ddp-workspace/projects/ddp-angio/src/app/components/end-enroll/end-enroll.component.ts

```diff
@@ -112,36 +112,37 @@ import {
                         <div class="timeline right">
                             <div class="card WrapperTL">
                                 <div class="card-body p-4 WrapperTL">
-                                    <h4>Februrary 2020</h4>
-                                    <h4 class="mb-0">ASCproject is featured in Nature Medicine (The Angiosarcoma Project:
+                                    <h4>February</h4>
+                                    <h1 class="margin-top-0">2020</h1>
+                                    <h4> ASCproject is featured in Nature Medicine (<a class="mb-0" target="_blank" href="https://www.nature.com/articles/s41591-019-0749-z">The Angiosarcoma Project:
                                         enabling genomic and clinical discoveries in rare cancer through patient
-                                        partnered research).</h4>
+                                        partnered research</a>)
+                                    </h4>
                                 </div>
                             </div>
                         </div>
                         <div class="timeline left">
                             <div class="card WrapperTL">
                                 <div class="card-body p-4 WrapperTL">
-                                    <h4>November 2021</h4>
+                                    <h4>November</h4>
```

Comment:
this might need to be November 2020 as per source

Reply:
fixed